appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/ProjectToolbar.java

```diff
@@ -202,6 +195,51 @@ public void execute() {
     }
   }
 
+  // Login to the New Gallery
+  private static class LoginToGalleryAction implements Command {
+    @Override
+      public void execute() {
+      Ode.getInstance().getProjectService().loginToGallery(
+        new OdeAsyncCallback<RpcResult>(
+          "Barf") {
```

Comment:
Maybe a more appropriate message in OdeMessages?

Reply:
Oops. Fixed.